numpy/fft/_pocketfft.py

```diff
@@ -113,10 +138,14 @@ def fft(a, n=None, axis=-1, norm=None):
     axis : int, optional
         Axis over which to compute the FFT.  If not given, the last axis is
         used.
-    norm : {None, "ortho"}, optional
+    norm : {"backward", "ortho", "forward"}, optional
         .. versionadded:: 1.10.0
 
-        Normalization mode (see `numpy.fft`). Default is None.
+        Normalization mode (see `numpy.fft`). Default is "backward".
```

Comment:
It would be great to have 1-2 sentences here pointing out that the normalization signals where the normalization occurs in the fft-ifft pair.  And/or just making this slightly longer to provide some of the info from the general docs.

Reply:
ok